python main.py

```
@@ -0,0 +1,8 @@
+from telegram.ext import Application, CommandHandler
+
+async def start(update, context):
```

Ruff (0.14.14)

[warning] Unused function argument: `context`

(ARG001)